### ISSUE TITLE
Can't add tuya cover switch 

### ISSUE THREAD

**Issue by thanhhlqn** [rospogrigio/localtuya#342]
I have a tuya switch to control my rolling door 
When I add this one by local tuya to hass, after filling ID and key device, at ID entity I only see this value to choose "1 (value: 3)", nothing else. And then I submit, I can't control my rolling door through hass.
I don't know add this one by platform "switch" or "cover". But even "switch" or "cover", there's only "1 (value: 3)" to choose. I don't know how to add it.
Btw, if I add this switch by integration Tuya (not local tuya through Hacs), it can recognize it's Cover, and I can control my rolling door from hass (by cover card). But with this way, when I use the physic switch to control door, hass system can not know it's real-time situation, hass don't know if my door close or open or closing or opening

**Comment by ropmip** [comment]
It should probably work with localtuya if you add ID1 and choose the option: 1 / 2 / 3 as values to: open / stop / close.

**Comment by thanhhlqn** [comment]
Yeah thanks. I’ve just tried this solution a couple of hours and it worked!
But it still has a problem. I can’t turn down. Just up and stop buttons are available :(

**Comment by ropmip** [comment]
Maybe the app thinks it is on fully closed position, so it doesn't show down option. Use the manual controls and/or try another method to detect position (if timed then add more seconds for duration). It also might be your unit is using another value for up.

**Comment by stevehop** [comment]
> 
> 
> Yeah thanks. I’ve just tried this solution a couple of hours and it worked!
> But it still has a problem. I can’t turn down. Just up and stop buttons are available :(

Have you tried v3.1.0? My cover (curtain motor) which works perfectly in v3.1.0 also has the same problem as yours when updating to v3.2.0 where it will only open and stop, the close button is unavailable
